plugins/source/aws/resources/services/ssoadmin/instances_fetch.go

```diff
@@ -0,0 +1,22 @@
+package ssoadmin
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ssoadmin"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchSsoadminInstances(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	svc := meta.(*client.Client).Services().SSOAdmin
+	config := ssoadmin.ListInstancesInput{}
+	response, err := svc.ListInstances(ctx, &config)
```

Comment:
Does this API needs to be paginated like the others? It seems it supports that functionality

Reply:
I think there is no possibility for more than 1 instance to exist. I can paginate it. It raises another question but we can look at that in next PR.